include/iso_alloc_internal.h

```diff
@@ -169,6 +169,21 @@ using namespace std;
 #define LIKELY(x) __builtin_expect(!!(x), 1)
 #define UNLIKELY(x) __builtin_expect(!!(x), 0)
 
+#if ALLOC_SANITY
```

Comment:
Can you remove this line? memcpy/memset sanity should not depend on `ALLOC_SANITY` being enabled.